Remotion/Security/Core/Metadata/AccessTypeReflector.cs

```diff
@@ -132,7 +132,7 @@ private void AddAccessTypesFromAttribute (IEnumerable<MethodInfo> methodInfos, D
     {
       foreach (var methodInfo in methodInfos)
       {
-        var values = _permissionReflector.GetRequiredMethodPermissions (methodInfo.DeclaringType, MethodInfoAdapter.Create(methodInfo));
+        var values = _permissionReflector.GetRequiredMethodPermissions (methodInfo.DeclaringType!, MethodInfoAdapter.Create(methodInfo));
```

Comment:
Do we have a ticket about guarding DeclaringType get-access?

Reply:
There is [RM-7811](https://re-motion.atlassian.net/browse/RM-7811) for the Mixins project, but it could be made more generic to add guarded extension methods for DeclaringType, just like for FullName.

Reply:
@MichaelKetting should I do this?

Reply:
Huh, I thought I wrote an answer: yes, please (unless it starts to look like it's getting too big for a single task).